README.md

```diff
@@ -10,10 +10,34 @@ It's totally not the first approach of this kind. But: this one should be easily
 
 Currently, Wireshell  consists of the following basic commands:
 
-#### New
+#### Fresh installation
```

Comment:
That was not exactly what I meant, but anyway, it's just a detail and I'll merge (and maybe modify later)